docs/user/azure/customization.md

```diff
@@ -60,6 +62,8 @@ compute:
       type: Standard_DS4_v2
       osDisk:
         diskSizeGB: 512
+      zones:
+      - "1"
```

Comment:
let's not give users example of single AZ, because that's really bad. at least 2 is more appropriate.

Reply:
I've added zones "2" and "3" to the example.